src/utilities/gql.ts

```diff
@@ -121,6 +133,10 @@ export enum Queries {
   MERGE_REQUESTS = 'merge_request',
   MISSION_MODEL = 'mission_model_by_pk',
   MISSION_MODELS = 'mission_model',
+  PARAMETER_DICTIONARIES = 'parameter_dictionary',
+  PARCEL_BY_PK = 'parcel_by_pk',
```

Comment:
Do you mind changing this to just `PARCEL` to be consistent with the singular/plural convention?

Reply:
Did you tag the wrong line here? It's singular here.

Reply:
No, sorry. I mean it should be `PARCEL` instead of `PARCEL_BY_PK`. The idea of this enum is to lightly generalize the purpose of the query from the actual query string so that the query string can potentially change later on without having to necessarily change the name of the constant.

Reply:
> No, sorry. I mean it should be `PARCEL` instead of `PARCEL_BY_PK`. The idea of this enum is to lightly generalize the purpose of the query from the actual query string so that the query string can potentially change later on without having to necessarily change the name of the constant.

Oh I see! Sorry I misunderstood. Yeah I'll change it.